manifests/params.pp

```diff
@@ -130,14 +130,6 @@
           'log_mode'    => '0755',
           'run_dir'     => '/run/nginx',
         }
-      } else {
```

Comment:
Why drop this? Doesn't Ubuntu 18.04 hit this? I also think it'd be a better and safer thing to have an `else` clause. If anything, Xenial should perhaps not manage the repo at all.

Reply:
the syntax isn't perfect here, above this block is one explicitly for Debian 9/10 and Ubuntu 18

Reply:
Still, having a safe default feels better to me. I'd also be in favor of dropping `manage_repo => true` and moving to not managing it by default. Then we can make the else block the future proof one so Debian 11/Ubuntu 20.04 are more likely to just work.

Reply:
I'm not sure if we should change any default here. However I'm fine with providing a default block that catches newer ubuntu/debian version to be future-proof. My personal preference is also to always use the upstream repositories and not the packages from Ubuntu.